include/obake/polynomials/polynomial.hpp

```diff
@@ -700,7 +699,14 @@ inline void poly_mul_impl_mt_hm(Ret &retval, const T &x, const U &y, const Args
             };
 
             using ::obake::detail::type_c;
+#if defined(_MSC_VER) && (_MSVC_LANG > 201703L)
+            auto vd1 = sorter(v1, type_c<T>{}, vseg1);
+            auto vd2 = sorter(v2, type_c<U>{}, vseg2);
+            return [vd1,vd2,
```

Comment:
I tried it with the std::move instead of copies but it doesn't work. It complains not being able to find vd2 which works just fine when the language version is set to C++17. Seems to me that a fix that some time ago was made to the C++17 version hasn't migrated into the C++20 (std:c++latest actually), yet. I haven't been able to reproduce the problem by using simple examples. All the trials I attempted worked fine.

Reply:
Ok fair enough, thanks for checking. I think we can leave the workaround as it is for the time being, but we should revisit it once the new MSVC matures.

Reply:
I intend to revisit all these issues with all new releases. One shouldn't forget that C++20 in VC is not officially release in any form, how could it? The standard is not official, yet.
For this specific one I expect it to be it just to move all the fixes from the C++17 parser into the C++20 parser when they are ready to.